web_src/js/webcomponents/overflow-menu.js

```diff
@@ -127,6 +127,26 @@ window.customElements.define('overflow-menu', class extends HTMLElement {
   });
 
   init() {
+    // for horizontal menus where fomantic boldens active items, prevent this bold text from
+    // enlarging the menu's active item replacing the text node with a div that renders a
+    // invisible pseudo-element that enlarges the box.
+    if (this.matches('.ui.secondary.pointing.menu') || this.matches('.ui.tabular.menu')) {
+      for (const item of this.querySelectorAll('.item')) {
+        for (const child of item.childNodes) {
+          if (child.nodeType === Node.TEXT_NODE) {
+            const text = child.textContent.trim();
```

Comment:
I think the "textContent" should be used as-is.

Because IIRC if the HTML is `<xxx/> text`, then the `textContent == " text"`, the space has a meaning there.

Reply:
Inside a flexbox, HTML whitespace has no meaning and I think these menus are all flexbox now. If not, then I agree it should render with whitespace.

Reply:
No problem, then maybe it could add a comment that "it only works for flex which doesn't render prefix/trailing spaces"

Reply:
`if (trim)` is needed to avoid creation of unnecessary empty text nodes.

Without the trim, it is a big ugly in the HTML:

<img width="436" alt="image" src="https://github.com/go-gitea/gitea/assets/115237/06a4e168-78a4-4baf-a80f-aa4a4c31f60a">